kalite/__init__.py

```diff
@@ -7,4 +7,11 @@
 __version__ = VERSION
 
 # ROOT_DATA_PATH is *not* where the source files live. It's a place where non-user-writable data files may be written.
-ROOT_DATA_PATH = os.path.join(sys.prefix, 'share', 'kalite')
+ROOT_DATA_PATH = None
```

Comment:
Sure. I guess I was wearing my C hat when I did this, since it felt weird to me not to have a common place to initialize this variable outside of the if branches, but it's probably fine in python. I'll change that.